internal/acceptance/template-strings-repository-docker.go

```diff
@@ -41,6 +47,9 @@ resource "nexus_repository_docker_proxy" "acceptance" {
 {{- end }}
 {{- if .Docker.HTTPSPort }}
 		https_port = "{{ .Docker.HTTPSPort }}"
+{{- end }}
+{{- if .Docker.Subdomain }}
+		https_port = "{{ .Docker.Subdomain }}"
```

Comment:
the tests can not working correct. you overwrite the variable https_port in your test string template instead of adding subdomain
```suggestion
		subdomain = "{{ .Docker.Subdomain }}"
```

Reply:
Derp. Great catch! Thank you. Definitely your suggestion is correct. I'm not sure why when I ran the test it wasn't caught.